velox/exec/tests/TableScanTest.cpp

```diff
@@ -333,16 +333,14 @@ TEST_P(TableScanTest, constDictLazy) {
        makeFlatVector<int64_t>(size, [](auto row) { return row; }),
        makeMapVector<int64_t, double>(
            size,
-           [](auto row) { return row % 3; },
+           [](auto row) { return row % 3 + 1; },
```

Comment:
Why is this change? Can you not write empty maps into DuckDB? If that's the case, let's file a GitHub issue with DuckDB.

Reply:
DuckDB requires use of the EMPTYLIST API which needs the DuckDB LogicalType for each Velox type. I didn't find any straightforward way to do so in Velox quite yet.

//! Create an empty list with the specified child-type
DUCKDB_API static Value EMPTYLIST(LogicalType child_type);

I didn't see the use of EMPTYLIST in Array etc either.

Are you familiar with anything for this ? I'm unsure if this is a gap on our side or theirs.

Reply:
Hmm... I can take a look tomorrow. @Mytherin Mark, would you know this off the top of your head?

Reply:
@aditi-pandit I think you can use fromVeloxType from velox/duckdb/conversion/DuckConversion.h to get LogicalType and pass that to Value::EMPTYLIST.

Reply:
This has only scalar types. But that's good enough for now. Will update the code.